tests/test_taling_a_vacation.py

```diff
@@ -0,0 +1,51 @@
+import sys
+
+import pytest
+
+from codecademy.taling_a_vacation import hotel_cost
+from codecademy.taling_a_vacation import plane_ride_cost
+from codecademy.taling_a_vacation import rental_car_cost
+from codecademy.taling_a_vacation import trip_cost
+
+@pytest.mark.parametrize("test_hotel_cost_input,test_hotel_cost_expected", [
+    (-sys.maxsize - 1, (-sys.maxsize - 1)*140),
+    (-sys.maxsize, -sys.maxsize*140),
+    (-1, -140),
+    (0, 0),
+    (1, 140),
+    (3037000499, 425180069860),
+    (sys.maxsize - 1, (sys.maxsize-1)*140),
+    (sys.maxsize, sys.maxsize*140)
```

Comment:
Also you do need some of the negative inputs - but must expected some exception. Hint: you have to take care of the invalid inputs in the function - yes it's a bug there and you have to fix it :)